src/Raven.Server/Documents/Handlers/Debugging/DatabaseDebugInfoPackageHandler.cs

```diff
@@ -31,7 +33,15 @@ public async Task GetInfoPackage()
                             { "database",new Microsoft.Extensions.Primitives.StringValues(Database.Name) }
                         };
 
-                        foreach (var route in DebugInfoPackageUtils.Routes.Where(x => x.TypeOfRoute == RouteInformation.RouteType.Databases))
+                        var routes = DebugInfoPackageUtils.Routes.Where(x => x.TypeOfRoute == RouteInformation.RouteType.Databases);
+                        if (Server.Certificate.Certificate != null)
+                        {
+                            var feature = HttpContext.Features.Get<IHttpAuthenticationFeature>() as RavenServer.AuthenticateConnection;
+                            Debug.Assert(feature != null);
+                            routes = routes.Where(route => feature.CanAccess(Database.Name, route.AuthorizationStatus != AuthorizationStatus.ValidUser));
```

Comment:
Shouldn't we pass `route.AuthorizationStatus == AuthorizationStatus.DatabaseAdmin` as the second parameter?

Reply:
I found that the code is different on 5.3 branch. We already have `DebugInfoPackageUtils.GetAuthorizedRoutes` there. Let's backport and use it here as we already do on 5.3 branch.

CC @haludi

Reply:
Ended up refactoring TryAuthorize and used that instead of CanAccess. Inside the function CanAccessRoute there is a correct use of DatabaseAdmin also

Reply:
Great